decentraland/unity-renderer
# fix: tutorial blocks on microphone step

## What does this PR change?

Fixes #2966 

## How to test the changes?

1. Open a fresh tab of Decentraland as guest
2. Go through the tutorial
3. When getting to the microphone step, press T. It should move the tutorial forward and not get stuck
4. Restart the tutorial as a wallet user and recheck step 3.
5. Restart the tutorial and check that if you open and close the friends window, it should move forward too.
6. Same as 5, but open the chat window and go to channel selection tab (press back on the #nearby titlte)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
